teamscale-client/src/main/java/com/teamscale/client/HttpUtils.java

```diff
@@ -102,24 +102,34 @@ private static void setUpProxyServer(OkHttpClient.Builder httpClientBuilder) {
 
 	private static boolean setUpProxyServerForProtocol(ProxySystemProperties.Protocol protocol,
 													   OkHttpClient.Builder httpClientBuilder) {
+		TeamscaleProxySystemProperties teamscaleProxySystemProperties = new TeamscaleProxySystemProperties(protocol);
```

Comment:
Why is Teamscale fine with the dependency on `ProxySystemProperties` one line below? Not sure what to do with those findings. We could either move the TeamscaleProxySystemProperties` into the utils in the architecture or do a bigger refactoring. Or ignore it :man_shrugging:

Reply:
It looks like the client component in https://cqse.teamscale.io/architecture/editor/teamscale-jacoco-agent/teamscale-jacoco-agent.architecture?t=ts%2F37747_proxy_password_file_option%3AHEAD contains the ProxySystemProperties on the feature branch, but not in master. I have no idea why this architecture is the way it is. @DreierF Do you know something about this? I would assume we can ignore this finding.

Reply:
The intention of the architecture here is that the utils component should be used from the client, but utils should not depend on classes from within the client. Not sure where the difference in the architecture definition on both branches comes from, but on your branch the ProxySystemProperties seems to have been added to the Utils component, which "solves the issue". So you could also define that TeamscaleProxySystemProperties belongs to Utils on master (i.e. after the ticket is merged) to fix the issue.